cli/src/pcluster/validators/directory_service_validators.py

```diff
@@ -78,3 +78,39 @@ def _validate(self, ldap_tls_reqcert):
                 f"For security reasons it's recommended to use {' or '.join(values_requiring_cert_validation)}",
                 FailureLevel.WARNING,
             )
+
+
+class AdditionalSssdConfigsValidator(Validator):
+    """AdditionalSssdConfigs validator."""
+
+    def _validate(self, additional_sssd_configs, ldap_access_filter):
+        """Validate that AdditionalSssdConfigs does not introduce unacceptable values."""
+        if "ldap_schema" in additional_sssd_configs:
+            value = additional_sssd_configs["ldap_schema"]
```

Comment:
minor: I'd use a more explicative term other than "value". 
Even because we're using the same word value in the next case and it can hide issues if we miss to set the right "value" to this variable.

I'd use `ldap_schema` or `actual_ldap_schema`.

Reply:
Fixed in the next rev